modules.yml

```diff
@@ -96,6 +96,8 @@ modules:
     used_by_otel: true
   comp/otelcol/otlp/testutil:
     used_by_otel: true
+  comp/otelcol/status/def: default
+  comp/otelcol/status/impl: default
```

Comment:
do these need the following:
```
    used_by_otel: true
```

Reply:
Good qustion - no, those are only used in the embedded collector (aka in Agent code base), while `used_by_otel` indicates used in the OSS collector contrib code base